lesson04/assignment/index.html

```diff
@@ -4,16 +4,29 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <link rel="stylesheet" href="/css/style.css">
-  <style></style>
-  <title>Document</title>
+  <link rel="stylesheet" href="css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700,700i&display=swap" rel="stylesheet">
+  <title>Transitions and Animations</title>
 </head>
 <body>
-  <div id="one">
-    <div id="two">
-
+  <main class="main">
+    <div class="card">
+      <div class="front">
+        <h1>Los Angeles</h1>
+      </div>
+      <div class="back">
+        <h1>Agenda</h1>
+        <ul class="back__ul"> <!-- Implemented this naming convention to apply the BEM methodology in the CSS file -->
+          <li class="back__ul--list">Breakfast:</li>
```

Comment:
For BEM, you'd want to use a single dash after ul so `back__ul-list`or `back__item`